scanner/datastore/postgres/distributions.go

```diff
@@ -2,15 +2,22 @@ package postgres
 
 import (
 	"context"
+	"fmt"
+	"regexp"
 
 	"github.com/quay/claircore"
+	"github.com/quay/zlog"
 )
 
+var rhelCPE = regexp.MustCompile(`cpe:2\.3:o:redhat:enterprise_linux:(\d+):\*:\*:\*:\*:\*:\*:\*`)
+
 // Distributions retrieves the currently known distributions from the database.
 //
 // A distribution is considered known if there exists at least one row in the vuln table which references it.
 func (m *matcherStore) Distributions(ctx context.Context) ([]claircore.Distribution, error) {
-	const selectDists = `SELECT DISTINCT dist_id, dist_version_id, dist_version FROM vuln`
+	ctx = zlog.ContextWithValues(ctx, "component", "datastore/postgres/distributions/Distributions")
+
+	const selectDists = `SELECT DISTINCT dist_id, dist_version_id, dist_version, repo_name FROM vuln WHERE repo_name = '' OR repo_name LIKE 'cpe:2.3:o:redhat:enterprise_linux:%:*:*:*:*:*:*:*'`
```

Comment:
Nit: A short comment on the repo_name matching could make things easier to read.

Reply:
https://github.com/stackrox/stackrox/pull/12452/commits/b94446c77b0bff7f966050d8cbee876094ca1ba2